src/prefect/cli/deploy.py

```diff
@@ -186,8 +186,10 @@ async def deploy(
         False,
         "--ci",
         help=(
-            "Run this command in CI mode. This will disable interactive prompts and"
-            " will error if any required arguments are not provided."
+            "DEPRECATED: Please use the global '--no-prompt' flag instead: 'prefect"
+            " --no-prompt deploy.\n\n Run this command in CI mode. This will disable"
```

Comment:
```suggestion
            " --no-prompt deploy'.\n\n Run this command in CI mode. This will disable"
```

Reply:
@AmanSal1 the output is missing the single quote still. See the above suggested change.

Reply:
@serinamarie Done with it

Reply:
Great, can you update the screenshot? I'm going to run the tests now.

Reply:
@serinamarie In the PR description? I have already done it

Reply:
Yeah, post-fix :)

Reply:
@serinamarie So do I have to change it or is it alright ? The description

Reply:
Don't worry about it; it just won't be completely accurate but it's a small thing.